src/bda/plone/shop/locales/de/LC_MESSAGES/bda.plone.shop.po

```diff
@@ -259,7 +259,7 @@ msgstr "Artikel mit UID {uuid} existiert nicht."
 #. Default: "Cart Discount"
 #: ./browser/navigation.py:607
 msgid "cart_discount"
-msgstr "Warenkorb Rabatt"
+msgstr "Rabatt"
```

Comment:
There's actually two types of discount

* Cart Discount - which scopes all items in the cart
* Item discount - which scopes to a set of or a specific item

Reply:
Yes, I'm aware of both `cart_discount` and `item_discount`; for the latter, there is the translation `Artikelrabatt`. "Rabatt" commonly means a discount on the total sum (I'm not aware of any exceptions); we wouldn't say "Warenkorb Rabatt", not even "Warenkorb-Rabatt".  (In German, we use hyphens when chaining words; "`Warenkorb Rabatt`" would be wrong anyway.)